open-craft/opencraft
# Create a BulkEmailFlag when first appserver is provisioned [SE-2078].

This automatically enables bulk emails for new instances. The playbook that enables bulk emails is run when provisioning the first appserver for the instance.

Note that this doesn't modify the default bulk email template, which is an edX-branded template.

**Test instructions**:
1. Checkout this code and create a new instance. Spawn the first appserver.
2. When the first appserver is successfully spawned, go to the django admin on the new appserver, and verify that a `BulkEmailFlag` with `enabled` attribute set to `True` and `required_course_email_auth` set to `False`. You may also want to verify that you can see the Email tab in the Instructor dashboard in the LMS.
3. Spawn another appserver. Verify that after the appserver spawns successfully the `BulkEmailFlag` remains intact and no new `BulkEmailFlag` object is created.

**Reviewers**:
